lib/elm-test.js

```diff
@@ -150,23 +98,24 @@ function main() {
       'Use a custom path to an Elm executable (default: elm)',
       undefined
     )
-    .option(
-      '--seed <int>',
-      'Run with a previous fuzzer seed',
-      parsePositiveInteger('--seed <int>'),
-      Math.floor(Math.random() * 407199254740991) + 1000
+    .addOption(
+      new Option('--seed <int>', 'Run with a previous fuzzer seed')
+        .default(Math.floor(Math.random() * 407199254740991) + 1000, 'random')
+        .argParser(parsePositiveInteger)
```

Comment:
Out of interest what is going here? The `'random'` looks a bit special.

Reply:
Now, the helps looks like:

```
  --seed <int>       Run with a previous fuzzer seed (default: random)
```

Previously, it was:

```
  --seed <int>                   Run with a previous fuzzer seed (default: 390695695167635)
```

With a different number each time.

So it wasn’t clear if the default seed was a constant or what it was.

Reply:
Nice!